html/signup.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="ko">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>판다마켓 - 로그인</title>
+  <meta name="description" content="판다와 함께 더 나은 서비스와 향상된 쇼핑생활을 만끽하세요" />
+  <meta property="og:image" content="../resources/images/img_home_bottom.png" />
+  <meta property="og:title" content="" />
+  <meta property="og:description" content="" />
+  <meta property="og:url" content="" />
+  <meta property="og:type" content="website" />
+  <meta name="twitter:card" content="summary_large_image" />
+  <meta name="twitter:image" content="../resources/images/img_home_bottom.png" />
+  <meta name="twitter:title" content=" " />
+  <meta name="twitter:description" content="" />
+  <link rel="shortcut icon" href="../resources/images/common/favicon.ico" type="image/x-icon" />
+  <link rel="icon" href="../resources/images/common/favicon.ico" type="image/x-icon" />
+  <link rel="preload" href="../resources/css/main.css" as="style" onload="this.onload=null;this.rel='stylesheet'" />
+</head>
+<body>
+  <div class='wrap'>
+    <div class='login_container'>
+      <h2 class='log_tit' tabindex="0"><img src="../resources/images/common/logo_tit.png" alt="판다마켓"></h2>
+      <div class="form_wrap">
+        <form>
+          <fieldset>
+            <legend>이메일/비밀번호 입력</legend>
+            <div class="input-container">
+              <label for='email'>이메일</label>
+              <input type="email" id='email' required autocomplete="email" placeholder='이메일을 입력해 주세요'>
+            </div>
+            <div class="input-container">
+              <label for='nickname'>닉네임</label>
+              <input type="text" id='nickname' placeholder='닉네임을 입력해 주세요'>
+            </div>
+            <div class="input-container">
+              <label for="password">비밀번호</label>
+              <input type="password" id="password" autocomplete="current-password" placeholder='비밀번호를 입력해 주세요'>
+              <button type="button" class="toggle-password hide-password-icon" data-target="password"></button>
+            </div>
+            <div class="input-container">
+              <label for="confirmPassword">비밀번호 확인</label>
+              <input type="password" id="confirmPassword" autocomplete="current-password" placeholder='비밀번호를 입력해 주세요'>
+              <button type="button" class="toggle-password hide-password-icon" data-target="confirmPassword"></button>
+              <div id="error-message" class="error-message">비밀번호가 일치하지 않습니다</div>
+            </div>
+          </fieldset>
+        </form>
+        <div class="btn_set">
+          <button type="button" class="btn_login" title="선택됨">로그인</button>
+        </div>
```

Comment:
❗️ 수정요청
해당 로그인 버튼이 제출될때 필요한 데이터들은 form 태그안에 있는 input에 사용자가 입력한 값입니다.
이러한 경우 button의 type은 submit이 더 적절합니다.
또한 기본적으로는 form 태그안에 button이 들어가는 것이 더 좋습니다~
그래야 form에서 제출 이벤트가 발생하면, type이 submit인 버튼의 onclick 이벤트가 실행되게 됩니다.
이를 더 풀어서 말하자면 form 태그안의 input, button 과 같은 상호작용 요소에서 submit 이벤트, enter key 입력이 발생하면 해당 form 에서의 submit 이벤트가 실행된다는 것입니다.

만약 디자인 상이나 여러가지 이유로 form 태그 밖에 button 태그가 존재해야 한다면 [button form](https://developer.mozilla.org/ko/docs/Web/HTML/Element/button#form) 속성을 통해 연결된 form 태그를 명시해주세요.

```suggestion
        <div class="btn_set">
          <button type="submit" form="formId" class="btn_login" title="선택됨">로그인</button>
        </div>
```